### ISSUE TITLE
EPIWEEK does not equal ISOWEEK

### ISSUE THREAD

**Issue by etheleon** [etheleon/dengue#32]
As discussed, we are using CDC WEEK which starts on a Sunday. No standard func in SQL to calculate that closest is ISOWEEK 
See comment in https://github.com/tidyverse/lubridate/issues/564

> Epidemiological weeks begin on Sunday in the U.S. and are used by the CDC and various state agencies. But in most of the rest of the world, epi weeks begin on Monday and are therefore equivalent to iso weeks. The World Health Organization and pretty much every other international organization, and most countries other than the US (as far as I know) use this ISO standard for epidemiological weeks.

```R
> epiweek("1987-12-28")
[1] 52
> isoweek("1987-12-28")
[1] 53
```

**Comment by etheleon** [comment]
Closed by #60

**Comment by etheleon** [comment]
Closed by #60
